src/scenarios/miner_std.py

```diff
@@ -7,13 +7,20 @@
 
 
 def cli_help():
-    return "Generate blocks over time. Options: [--allnodes | --interval=<number>]"
+    return "Generate blocks over time. Options: [--allnodes | --interval=<number> | --mature ]"
 
+class Miner:
+    def __init__(self, node, mature):
+        self.node = node
+        self.wallet = ensure_miner(self.node)
+        self.addr = self.wallet.getnewaddress()
```

Comment:
Really glad to finally see this abstracted away and only called once. been meaning to do that for ages!